robot_ws/src/voice_interaction_robot/nodes/integration_test

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python
+"""
+ Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+ Permission is hereby granted, free of charge, to any person obtaining a copy of this
+ software and associated documentation files (the "Software"), to deal in the Software
+ without restriction, including without limitation the rights to use, copy, modify,
+ merge, publish, distribute, sublicense, and/or sell copies of the Software, and to
+ permit persons to whom the Software is furnished to do so.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+ INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+ PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+"""
+
+
+"""
+    This script runs a series of integration tests for the voice_interaction node. 
+    It can be launched with the integration_test launch file and it will output
+    how many tests are to be run and the results for each test. 
+"""
+
+import rospy
+import rosnode
+import time
+from std_msgs.msg import String
+from geometry_msgs.msg import Twist, Vector3
+
+text_input_publisher = rospy.Publisher("/text_input", String, queue_size=5)
```

Comment:
Global creation of a publisher before initializing a node would result in an error. To prevent this, either create the publishers after the init_node or put them in the VoiceInteractionIntegration test class.
https://answers.ros.org/question/269630/relationship-between-rospyinit_node-and-rospypublisher/